### PR TITLE
Change MIME type references to Media (MIME) type

### PR DESCRIPTION
**Pull request by MauricioFauth**
### Description

Change _MIME type_ references to _Media (MIME) type_.

Related to #15350

### CODE REVIEW

**Review by williamdes — APPROVED** [review]

File: doc/glossary.rst [hunk 1]
```diff
@@ -220,6 +220,12 @@ From Wikipedia, the free encyclopedia
 
       .. seealso:: <https://secure.php.net/mcrypt>
 
+    Media type
+      A media type (formerly known as MIME type) is a two-part identifier
+      for file formats and format contents transmitted on the Internet.
+
+      .. seealso:: <https://en.wikipedia.org/wiki/Media_type>
+
```
Comment (ibennetch) [inline doc/glossary.rst]: 👍

File: libraries/classes/Plugins/Export/ExportHtmlword.php [hunk 2]
```diff
@@ -412,7 +412,7 @@ public function getTableDef(
         }
         if ($do_mime && $cfgRelation['mimework']) {
             $schema_insert .= '<td class="print"><strong>'
-                . htmlspecialchars('MIME')
+                . __('Media type (MIME)')
```
Comment (ibennetch) [inline libraries/classes/Plugins/Export/ExportHtmlword.php]: The original use of `htmlspecialchars()` is odd to me here, but this seems like a good change.

File: libraries/classes/Plugins/Export/Helpers/Pdf.php [hunk 3]
```diff
@@ -554,7 +554,7 @@ public function getTableDef(
             $columns_cnt++;
         }
         if ($do_mime && $cfgRelation['mimework']) {
-            $this->colTitles[$columns_cnt] = __('MIME');
+            $this->colTitles[$columns_cnt] = __('Media type (MIME)');
```
Comment (ibennetch) [inline libraries/classes/Plugins/Export/Helpers/Pdf.php]: In several places, you flip back and forth between "Media type (MIME)" and "Media (MIME) type". We should pick one and I think the second way sounds more natural. Therefore, I'd change this one.

File: templates/database/data_dictionary/index.twig [hunk 4]
```diff
@@ -22,7 +22,7 @@
           {% endif %}
           <th>{% trans 'Comments' %}</th>
           {% if table.has_mime %}
-            <th>MIME</th>
+            <th>{% trans 'Media type (MIME)' %}</th>
```
Comment (ibennetch) [inline templates/database/data_dictionary/index.twig]: And the order here as well.

**Review by ibennetch — CHANGES_REQUESTED** [review]
Nice work. I left a few comments in line.

**Comment by mauriciofauth** [comment]
@ibennetch Done! I fixed the inconsistencies.

**Pull request merged**